elements/campaign-display/campaign-display.test.js

```diff
@@ -12,7 +12,7 @@ describe('<campaign-display>', () => {
   let crop;
   let campaign;
   beforeEach(() => {
-    // TODO: Prevent setState warnings spamming the console
+    // Prevent setState warnings spamming the console
```

Comment:
Leave `TODO` as an obvious marker that we need to come back to this. If we're not planning on that, remove either this comment, the one below, or both.